aeon_internal.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "aeon_error.proto";
+import "aeon_value.proto";
+
+package aeon;
+
+// Internal API to Aeon - a distributed database based on Tarantool.
+service InternalService {
+  // Get configurations of all instances.
+  rpc GetConfig(GetConfigRequest) returns (GetConfigResponse) {}
+}
+
+// Get configurations of all instances.
+
+message GetConfigRequest {}
+
+message GetConfigResponse {
+  // Error information. Set only on failure.
+  Error error = 1;
+  // Configurations of instances.
+  Value config = 2;
```

Comment:
Describing an instance config as `Value` (or rather `MapValue`) is fine but we should probably add some structure here to describe fields we add to the return value (instance name, replicaset name, etc).

Reply:
Thanks, fixed.